data/data-pipeline/data_pipeline/etl/score/constants.py

```diff
@@ -194,88 +194,84 @@
     field_names.SCORE_L_COMMUNITIES,
     field_names.TOTAL_POP_FIELD,
     field_names.FPL_200_SERIES,
-    field_names.POVERTY_LESS_THAN_200_FPL_FIELD,
-    field_names.POVERTY_LESS_THAN_200_FPL_FIELD
-    + field_names.PERCENTILE_FIELD_SUFFIX,
-    field_names.EXPECTED_AGRICULTURE_LOSS_RATE_FIELD,
+    field_names.EXPECTED_AGRICULTURE_LOSS_RATE_LOW_INCOME_FIELD,
     field_names.EXPECTED_AGRICULTURE_LOSS_RATE_FIELD
     + field_names.PERCENTILE_FIELD_SUFFIX,
-    field_names.EXPECTED_AGRICULTURE_LOSS_RATE_LOW_INCOME_FIELD,
-    field_names.EXPECTED_BUILDING_LOSS_RATE_FIELD,
+    field_names.EXPECTED_AGRICULTURE_LOSS_RATE_FIELD,
+    field_names.EXPECTED_BUILDING_LOSS_RATE_LOW_INCOME_FIELD,
     field_names.EXPECTED_BUILDING_LOSS_RATE_FIELD
     + field_names.PERCENTILE_FIELD_SUFFIX,
-    field_names.EXPECTED_BUILDING_LOSS_RATE_LOW_INCOME_FIELD,
-    field_names.EXPECTED_POPULATION_LOSS_RATE_FIELD,
+    field_names.EXPECTED_POPULATION_LOSS_RATE_LOW_INCOME_FIELD,
     field_names.EXPECTED_POPULATION_LOSS_RATE_FIELD
     + field_names.PERCENTILE_FIELD_SUFFIX,
```

Comment:
Seems like we're missing `field_names.EXPECTED_POPULATION_LOSS_RATE_FIELD`. 

I think every field needs to have: 

* BLAH_LOW_INCOME_FIELD (or for the workforce category, BLAH_LOW_HS_eDUCATION_FIELD) 
* BLAH + field_names.PERCENTILE_FIELD
* BLAH (the raw value)

Reply:
Done.